source/connect/mongoclient.txt

```diff
@@ -0,0 +1,120 @@
+.. _kotlin-sync-mongoclient:
+
+====================
+Create a MongoClient
+====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and perform 
+  operations on the MongoDB deployment.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoClient`` object
+to connect to MongoDB.
+
+.. _kotlin-sync-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. TODO, add this as last sentence for ``username:password`` description: For more information about the ``authSource`` 
```

Comment:
Does this TODO need to be done?

Reply:
A reminder for when the kotlin authentication page is created. Made comment clearer, thanks!